website/docs/configuration/changelog_examples.md

```diff
@@ -0,0 +1,440 @@
+]
+
+link_parsers = [
+  { pattern = "#(\\d+)", href = "{{ remote.link }}/issues/$1" },
+  { pattern = "RFC(\\d+)", text = "ietf-rfc$1", href = "https://datatracker.ietf.org/doc/html/rfc$1" },
+]
+```
+
+</details>
+
+<details>
+  <summary>Raw Output</summary>
+
+```md
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### ⛰️  Features
+
+- feat(config): support multiple file formats ([a9d4050](a9d4050212a18f6b3bd76e2e41fbb9045d268b80))
+- feat(cache): use cache while fetching pages ([df6aef4](df6aef41292f3ffe5887754232e6ea7831c50ba5))
+
+## [1.0.1](https://github.com/orhun/git-cliff-readme-example/compare/v1.0.0...v1.0.1)
+
+### 🚜 Refactor
+
+- refactor(parser): expose string functions ([e4fd3cf](e4fd3cf8e2e6f49c0b57f66416e886c37cbb3715))
+
+### ⚙️ Miscellaneous Tasks
+
+- chore(release): add release script ([06412ac](06412ac1dd4071006c465dde6597a21d4367a158))
+
+## [1.0.0] - 2021-07-18
+
+### ⛰️  Features
+
+- feat(parser): add ability to parse arrays ([a140cef](a140cef0405e0bcbfb5de44ff59e091527d91b38))
+
+### 🐛 Bug Fixes
+
+- fix(args): rename help argument due to conflict ([9add0d4](9add0d4616dc95a6ea8b01d5e4d233876b6e5e00))
+
+### 📚 Documentation
+
+- docs(project): add README.md ([81fbc63](81fbc6365484abf0b4f4b05d384175763ad8db44))
+- docs(example)!: add tested usage example ([ad27b43](ad27b43e8032671afb4809a1a3ecf12f45c60e0e))
+```
+
+</details>
+
+<details>
+  <summary>Rendered Output</summary>
+
+```mdx-code-block
+# Changelog
+All notable changes to this project will be documented in this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+## [Unreleased]
+### ⛰️  Features
+- feat(config): support multiple file formats (a9d4050)
+- feat(cache): use cache while fetching pages (df6aef4)
+## [1.0.1](https://github.com/orhun/git-cliff-readme-example/compare/v1.0.0...v1.0.1)
+### 🚜 Refactor
+- refactor(parser): expose string functions (e4fd3cf)
+### ⚙️ Miscellaneous Tasks
+- chore(release): add release script (06412ac)
+## [1.0.0] - 2021-07-18
+### ⛰️  Features
+- feat(parser): add ability to parse arrays (a140cef)
+### 🐛 Bug Fixes
+- fix(args): rename help argument due to conflict (9add0d4)
+### 📚 Documentation
+- docs(project): add README.md (81fbc63)
+- docs(example)!: add tested usage example (ad27b43)
+```
+
+</details>
+
+## Detailed
+
+<details>
+  <summary>TOML configuration</summary>
+
+```toml
+[changelog]
+header = """# Changelog
+
+All notable changes to this project will be documented in this file.
+
+"""
+
+body = """
+{% if version %}\
+    ## [{{ version | trim_start_matches(pat="v") }}] - {{ timestamp | date(format="%Y-%m-%d") }}
+{% else %}\
```

Comment:
We never go into the `else` because release-plz always populates the `version` field. I'm removing the detailed template because it doesn't fit for release-plz. If needed, we can add it into another PR 👍

Reply:
sorry about that!

truth be told I have been struggliong a lot with testing out stuff because I had to rely on `git cliff -o CHANGELOG.md` during my contribution. I am yet to find a way (even for mty personal use) to generate a changelog from scratch entirely with release-plz. Is there a command to do that?

Reply:
>  to generate a changelog from scratch entirely with release-plz. Is there a command to do that?

no 😂 https://github.com/MarcoIeni/release-plz/issues/1009

> sorry about that!



Don't worry :)